lib/pf/ipset_cache.pm

```diff
@@ -0,0 +1,135 @@
+package pf::ipset_cache;
+
+=head1 NAME
+
+pf::ipset_cache -
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::ipset_cache
+
+=cut
+
+use strict;
+use warnings;
+use Moo;
+use CHI;
+use pf::log;
+use pf::util qw(pf_run);
+
+our $logger = get_logger();
```

Comment:
Yes, However for modules that are sub classes the better approach is to wrap getting the logger in a method.
To ensure we are getting the correct logger for that class.